File: uniswift/Uni/Uni/Declaration/DeclarationElement.swift
//
//  DeclarationElement.swift
//  Uni
//
//  Created by Michael Vlasov on 06.05.2018.
//  Copyright © 2018 Unified. All rights reserved.
//

import Foundation

public struct DeclarationElement {
	public let attributes: [DeclarationAttribute]
	public let children: [DeclarationElement]
	
	public init(attributes: [DeclarationAttribute], children: [DeclarationElement]) {
		self.attributes = attributes
		self.children = children
	}
	
}

File: unijava/src/org/unified/uni/declaration/DeclarationElement.java
package org.unified.uni.declaration;

/**
 * Uni Declaration Element
 * <p>
 * Created by Michael Vlasov on 19.04.2018.
 * <p>
 * Copyright (c) 2018 Michael Vlasov
 */
public final class DeclarationElement {
	public final DeclarationAttribute[] attributes;
	public final DeclarationElement[] children;


	public DeclarationElement(DeclarationAttribute[] attributes, DeclarationElement[] children) {
		this.attributes = attributes;
		this.children = children;
	}


	public static final DeclarationElement[] emptyArray = new DeclarationElement[0];
}
